cloudzero/manifest.json

```diff
@@ -0,0 +1,30 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "6f1a61c2-e875-42f7-b8ba-94b191786846",
+  "app_id": "cloudzero",
+  "display_on_public_website": false,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "View and analyze your Datadog costs on the CloudZero platform.",
+    "title": "CloudZero",
+    "media": [],
```

Comment:
Please add images - we recommend adding screenshots of your Datadog dashboard and of your platform. Note our [media guidelines here](https://docs.datadoghq.com/developers/integrations/create_a_tile/?tab=buildatileontheintegrationspage#media-carousel), which require all images to have a 16:9 aspect ratio.